tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -111,9 +112,11 @@ func newStressDelete(kvc pb.KVClient, keySuffixRange int) stressFunc {
 
 func newStressDeletePrefix(kvc pb.KVClient, keySuffixRange int) stressFunc {
```

Comment:
These should probably be renamed to newStressRangeInterval / newStressDeleteInterval(kvc, keySuffixRange, limit) since they're not prefixes any more.